fs/driver/fs_blockpartition.c

```diff
@@ -366,14 +357,43 @@ int register_blockpartition(FAR const char *partition,
   ret = register_blockdriver(partition, &g_part_bops, mode, dev);
   if (ret < 0)
     {
-      goto errout_release;
+      goto errout_free;
     }
 
   return OK;
 
-errout_release:
-  inode_release(dev->parent);
 errout_free:
+  inode_release(parent);
   kmm_free(dev);
   return ret;
 }
+
```

Comment:
Please include function comments Name/Description comments.

Reply:
since both functions are similar, they share the same comment here:
https://github.com/apache/nuttx/pull/9347/files#diff-ab5dfc7f5f96322da067b0ef28ca492f679dfac924852c7442592feec6696f28R296-R306

Reply:
But a script to create Documentation from these comments will know understand it and will fail to generate documentation.

Reply:
The header file already has the separated comment:
https://github.com/apache/nuttx/pull/9347/files#diff-4adf39d8e004d272f60b0655b13a65d44b392602445954ff14c89f3aed12a230R78-R105

Reply:
My fault, I missed the header files.